controllers/chefs/getChefOrdersByStatus/index.js

```diff
@@ -0,0 +1,14 @@
+import { NotFoundError } from '../../../helpers/index.js';
+import Order from '../../../models/order/Order.model.js';
+
+export const getChefOrdersByStatus = async (req, res) => {
```

Comment:
This route already duplicates the logic of the route /orders/by-chef/:orderId?filter=status:<status_name>